contracts/smart-account/common/Stakeable.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {IEntryPoint} from "@account-abstraction/contracts/interfaces/IEntryPoint.sol";
+
+/**
+ * @title Stakeable Entity
+ * @author Fil Makarov - <filipp.makarov@biconomy.io>
+ */
+contract Stakeable is Ownable {
+    constructor(address _newOwner) {
+        _transferOwnership(_newOwner);
+    }
+
+    function addStake(
+        address epAddress,
+        uint32 unstakeDelaySec
+    ) external payable onlyOwner {
+        IEntryPoint(epAddress).addStake{value: msg.value}(unstakeDelaySec);
```

Comment:
What's your opinion on zero address checks? I know some auditors flag any functions that do not have zero address checks.

Reply:
I think it's better to do zero address checks, however, those methods are ownable, so not public. I don't think owner will pass zero_address.
However, since the methods won't be called often, I can add zero address check despite they spend gas